### ISSUE TITLE
Incorrect myalerts_js_popup template.

### ISSUE THREAD

**Issue by Sama34** [MyBBStuff/MyAlerts#155]
This is the code in my `myalerts_js_popup` template on a fresh install.
```HTML
<script type="text/javascript">
    var unreadAlerts = '{$mybb->user['
    unreadAlerts
    ']}';
</script>
<script type="text/javascript" src="{$mybb->asset_url}/jscripts/myalerts.js"></script>
```

This results on `unreadAlerts` with a value of `` always.

**Comment by euantorano** [comment]
Hi, the template is already fixed for 2.0.3: https://github.com/MyBBStuff/MyAlerts/blob/master/inc/plugins/MybbStuff/MyAlerts/templates/js_popup.html :smile:

**Comment by Sama34** [comment]
I might not be up to date.
